functions-and-scope.js

```diff
@@ -78,7 +149,23 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Zorg ervoor dat jouw functie ook werkt als we een andere array willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+let largest=0;
+function higestGrade(arrVal) {
+    for (let i = 0; i < arrVal.length; i++) {
+        if(arrVal[i]>largest){
+            largest = arrVal[i];
+        }
+    }
+    return largest;
+}
```

Comment:
In deze opdracht is het prima om als default waarde 0 te pakken, maar stel je eens voor dat hier een array binnenkomt als [-1, -4, -2, -3]. In dit voorbeeld is de hoogste waarde -1, maar met jouw functie zou er 0 uit komen terwijl die niet eens in de array voorkomt.

Nogmaals, het maakt voor deze situatie niet uit, maar het is goed om je te realiseren dat wanneer operaties als deze gaat doen op een array, dat je niet een default waarde pakt, maar altijd de eerste waarde van de array, dan weet je zeker dat je goed zit namelijk:
```suggestion
function higestGrade(arrVal) {
    let largest = arrVal[0];
    
    for (let i = 0; i < arrVal.length; i++) {
        if(arrVal[i]>largest){
            largest = arrVal[i];
        }
    }
    return largest;
}
```